test/spec/features/context-pad/ContextPadSpec.js

```diff
@@ -794,6 +802,7 @@ describe('features/context-pad', function() {
       contextPad.trigger('click', event);
 
       // then
+      expect(triggerSpy).to.have.been.calledOnce;
```

Comment:
Let's assert what it has been called with; potentially in a separate spec ("event integration").